phpstan.neon.dist

```diff
@@ -10,5 +10,9 @@ parameters:
         - src/Test/AbstractWidgetTestCase.php
         - tests/bootstrap.php
 
+    ignoreErrors:
+        - '#^Parameter \$securityContext of method Sonata\\IntlBundle\\Timezone\\UserBasedTimezoneDetector::__construct\(\) has invalid typehint type Symfony\\Component\\Security\\Core\\Authentication\\Token\\Storage\\TokenStorageInterface\.$#'
+        - '#^Class Sonata\\UserBundle\\Model\\User not found\.$#'
```

Comment:
There is a parameter you can use in the github action to install dependencies and avoid this.